components/EditForm/index.js

```diff
@@ -58,6 +59,13 @@ export default function EditForm({ project }) {
     ]);
   };
 
+  //delete new item
```

Comment:
I don't think that you need this comment because you've chosen a good function name.
```suggestion
```